src/filters/pages.js

```diff
@@ -291,8 +291,7 @@ const pageType = (tags) => {
     'Links',
     'News',
   ];
-
-  return tags ? tags.find((tag) => types.includes(tag)) : false;
+  return tags.filter((tag) => types.includes(tag));
```

Comment:
This change is causing the build to fail, because 1) it is expecting a `string` return value instead of an array, and 2) `tags` might be missing entirely.

```suggestion
  return tags ? tags.find((tag) => types.includes(tag)) : false;
```

Reply:
OK.  Does it make sense for this to return an empty string instead of false if the tag isn't included in the list? (I think I may have made that change and the build is successful.)

Reply:
Hm. I'm not sure practically, but in some ways `false` makes more sense to me theoretically. Or `undefined`?